config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml

```diff
@@ -140,6 +140,8 @@ periodics:
       - --scenario=kubernetes_e2e
       - --
       - --cluster=e2e-big-beta
+      # Don't override metrics-bind-address in 1.13
```

Comment:
This will get outdated as soon as we will have 1.15 branch :-)

Can you change that to a TODO in a form to get rid of it as soon as this is testing 1.14+ branch?